### PR TITLE
Italian translation for admin.php

### PR DESCRIPTION
**Pull request by agaplus**

### CODE REVIEW

File: modules/admin/src/messages/it/admin.php [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    'ngrest_crud_filter_prompt' => 'Seleziona Filtro',
+    'ngrest_crud_ajax_search_length' => 'Il termine di ricerca deve essere di almeno 3 caratteri.',
+    'menu_system_logger' => 'Log',
+    'layout_filemanager_detail_replace_file' => 'Sostituisci file',
+    'layout_rightbar_languagelabel' => 'Lingua',
+    'layout_rightbar_savebtn' => 'Salva',
+    'aws_changepassword_notequal' => 'La nuova password e la nuova password ripetuta non sono uguali.',
+    'aws_changeapssword_minchar' => 'La nuova password deve contenere almeno {min} caratteri.',
+    
+// 1.0.0
```
Comment (nadar) [inline modules/admin/src/messages/it/admin.php]: Those translations (from 1.0.0 and below) should be translated as well. Thank you very much!

**Comment by nadar** [comment]
Thank your the translations, this is great! I just made you an issues so you can see which files are missing. 

https://github.com/luyadev/luya/issues/1154

**Comment by agaplus** [comment]
Hello @nadar , i just push the file that you specify of cms module in italian. That's Ok?

**Comment by nadar** [comment]
Could you not please send the changes in this pull request? Cause the PR #1155 contains also other files you have already commited. In order to add the changes to this pull request:

> Add more commits by pushing to the patch-2 branch on agaplus/luya.

**Comment by agaplus** [comment]
And now? That's Ok?

**Comment by nadar** [comment]
@agaplus It seems that you are mixing the pull request and branches. I am going to pick the files from your repository so its less confusing for you and i can close the PRs. For future PRs you should maybe check the docs in order to understand that a PR is connected to a branch from your repository.

Thanks for the translations!

**Comment by agaplus** [comment]
Ok. I'm new on github and i'm learning. I hope I was helpful. See you later.

**Comment by nadar** [comment]
@agaplus Of course! Thank you. I just added the missing translations. https://github.com/luyadev/luya/issues/1154

**Pull request closed without merging**